### PR TITLE
blog post to announce 2.2.0

### PR DESCRIPTION
**Pull request by fiona-naughton**
Template blog post for version 2.2.0 release (currently largely empty, details tba)

### CODE REVIEW

File: __posts/2022-06-02-release-2.2.0.md [hunk 1]
```diff
@@ -30,15 +30,15 @@ For more help with installation see the [installation instructions in the User G
 
 ## Notable new additions
 
- - The `frames` argument was added to AnalysisBase-derived classes (i.e. modern analysis classes) allowing for specific frames to be defined when running an analysis. (PR #3415)
+ - The `frames` argument was added to AnalysisBase-derived classes (i.e. modern analysis classes) allowing for specific frames to be defined when running an analysis. This new feature allows one to run per-frame analysis on arbitrary subsets of trajectory frames. (PR #3415)
```
Comment (IAlibay) [inline __posts/2022-06-02-release-2.2.0.md:33]: We should add the user github names like we've been doing in previous release posts.

File: __posts/2022-06-02-release-2.2.0.md [hunk 2]
```diff
@@ -30,15 +30,15 @@ For more help with installation see the [installation instructions in the User G
...
  - Python wheels are now made available through PyPI for x86_64 architectures (Issue #1300, PR #3680)
```
Comment (IAlibay) [inline __posts/2022-06-02-release-2.2.0.md:34]: We should also highlight Tyler's work on getting M1 builds to work (and thank Mike for testing it on release).

File: __posts/2022-06-02-release-2.2.0.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ For a full list of bugfixes see the [CHANGELOG][].
```
Comment (IAlibay) [inline __posts/2022-06-02-release-2.2.0.md:87]: 30 contributors, 21 of which are new contributors

New contributors are:
BFedder, 
inomag, 
Agorfa, 
aya9aladdin, 
shudipto-amin
HenokB,
umak1106,
tamandeeps,
Mrqeoqqt,
megosato,
AnirG,
rishu235,
manishsaini6421,
Sukeerti1,
robotjellyzone,
markvrma,
alescoulie,
mjtadema,
aditi2906,
rsexton2,
rafaelpap,

File: __posts/2022-06-02-release-2.2.0.md (hunk 1 above)
Comment (orbeckst) [reply __posts/2022-06-02-release-2.2.0.md:36]: added

File: __posts/2022-06-02-release-2.2.0.md (hunk 2 above)
Comment (orbeckst) [reply __posts/2022-06-02-release-2.2.0.md:37]: added

File: __posts/2022-06-02-release-2.2.0.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+For a full list of bugfixes see the [CHANGELOG][].
+
+
+## Other changes
+
+ - The number of matches allowed when doing a smarts selection has been increased from the default
+   1000 to `max(1000, n_atoms * 10)`, an additional set of `smarts_kwargs` can now also be passed
+   to override this behaviour (Issue #3469, PR #3470)
+ - The `fasteners` package is now a core dependency (PR #3375)
+ - LinearDensity now saves the histogram bin edges for easier plotting as `hist_bin_edges for
```
Comment (orbeckst) [inline __posts/2022-06-02-release-2.2.0.md:76]:
```suggestion
 - LinearDensity now saves the histogram bin edges for easier plotting as `hist_bin_edges` for
```

File: __posts/2022-06-02-release-2.2.0.md (hunk 3 above)
Comment (orbeckst) [reply __posts/2022-06-02-release-2.2.0.md:90]: thanks for the stats, added

**Review by orbeckst — APPROVED** [review]
Looks good, thank you @fiona-naughton and @IAlibay for the post. If anyone (@jbarnoud ?) wants to sprinkle some emojis over it then please go ahead — I don't know ho to find them on my keyboard.

File: __posts/2022-06-02-release-2.2.0.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+We are happy to release the version 2.2.0 of MDAnalysis!
+
+In line with [NEP 29][], this version of MDAnalysis drops support for Python 3.7 and raises the minimum NumPy version to 1.19.0. Minimum version support has also been changed for the following packages; `networkx>=2.0`, `scipy>=1.5.0`, `gsd>=1.9.3`. Further details on MDAnalysis future support strategy and NEP29 will be released shortly.
+
+Supported Python versions: **3.8, 3.9, 3.10**
+
+Supported Operating Systems:
+  - **Linux**
+  - **Windows**
+  - **MacOS** (on [x86_64](https://en.wikipedia.org/wiki/X86-64) and for the first time on *[Apple M1](https://en.wikipedia.org/wiki/Apple_M1) ARM* architecture)
```
Comment (orbeckst) [inline __posts/2022-06-02-release-2.2.0.md:15]: Do we need to say that we don't have cf packages yet but that pip installs should work?

**Comment by orbeckst** [comment]
@IAlibay is this ready to be merged & tweeted?

**Comment by IAlibay** [comment]
> @IAlibay is this ready to be merged & tweeted?

I think the only thing I was maybe hoping we could do is add the relevant folks that did the notable pieces of work, but if that's too much effort / not worth it then we can skip it.

File: __posts/2022-06-02-release-2.2.0.md (hunk 1 above)
Comment (orbeckst) [reply __posts/2022-06-02-release-2.2.0.md:36]: You mean for each new thing? In which case I misunderstood you — I thought you only meant the "new contribs" list that was added based on your other comment.

At this stage having detailed features with names is a "nice to have".

**Comment by orbeckst** [comment]
Oh, just saw your comment — I think it would be nice but if nobody can find the time then we go ahead with what we have. Is there anything that's like a special GSOC or other project that's included and that could be highlighted?

File: __posts/2022-06-02-release-2.2.0.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+@markvrma
+@alescoulie
+@mjtadema
+@aditi2906
+@rsexton2
+@rafaelpap
+
+## Acknowledgements
+
+Special thanks to @tylerjereddy for getting the M1 builds to work and to @mikemhenry for testing them locally.
```
Comment (IAlibay) [inline __posts/2022-06-02-release-2.2.0.md:118]: Should this just be under notable additions?

**Comment by IAlibay** [comment]
Assigned to @fiona-naughton who will add changes & merge in a little while

**Pull request merged**